.github/workflows/land-blocking.yml

```diff
@@ -45,6 +40,24 @@ jobs:
           done
           echo "Build failed after retries"
           exit 1
+  build-and-run-cluster-test:
+    name: Build images and run cluster test
```

Comment:
FYI - this is the signal Homu uses to gate the land.  If we change it here, we have to make the corresponding change in Homu.

Reply:
That's why I did not change its name. The behaviour should still be the same since it depends on the build-job.